src/components/ProjectDetails.js

```diff
@@ -12,25 +12,29 @@ class ProjectDetails extends Component {
     if (this.props.data) {
       const technologies = this.props.data.technologies;
       const images = this.props.data.images;
+      const obj = {
+        'api': api1,
+      }
       var title = this.props.data.title;
       var description = this.props.data.description;
       var url = this.props.data.url;
       if (this.props.data.technologies) {
         var tech = technologies.map((icons, i) => {
```

Comment:
probably not related to your code, but I feel like "technologies" should be `technologyIcons` and "icons" should be `icon`? (we get one icon per technology in the map, right?)

Reply:
I think it is kind of related to my code, but in another PR. As mentioned there are commits from another PR also in this PR.

Reply:
makes sense!